### ISSUE TITLE
P-values are not captured in GWAS output

### ISSUE THREAD

**Issue by cmungall** [monarch-initiative/dipper#482]
Except as text. Obviously people will want to filter on p-values.

Also: GWAS missing from https://github.com/monarch-initiative/ingest-artifacts/tree/master/sources

**Comment by kshefchek** [comment]
We should add odds ratio as well.

**Comment by mbrush** [comment]
yes, we can get p-values in.  re: ingest artifacts - cmaps may be missing for many of the older sources that nicole did before i started working with her. or things that kent has done on his own.  as necessary, we can go back and create cmaps for these older sources (or whatever form of documentation we decide to move forward with).
